server/src/lobby/game_client.rs

```diff
@@ -0,0 +1,28 @@
+use std::{collections::VecDeque, time::Instant};
+
+use crate::game::{player::PlayerIndex, spectator::spectator_pointer::SpectatorIndex};
+
+#[derive(Clone, Debug)]
+pub struct GameClient{
+    pub client_location: GameClientLocation,
+    pub host: bool,
```

Comment:
It seems like host is stored in two spots -- here and on the player/spectator

Reply:
I realized that, Ill make an issue for it.